doc/release/release_0.20.rst

```diff
@@ -0,0 +1,479 @@
+Announcement: scikit-image 0.20.0
+=================================
+
+We're happy to announce the release of scikit-image v0.20.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+
+New features and improvements
+-----------------------------
+- Several footprint generating and consuming functions in ``skimage.morphology`` now support footprint decomposition.
+  By decomposing a footprint into several smaller ones, morphological operations can potentially be speed up.
+  The decomposed footprint can be generated with the new ``decomposition`` parameter of the functions ``rectangle``, ``diamond``, ``disk``, ``cube``, ``octahedron``, ``ball``, ``octagon`` in ``skimage.morphology``.
+  The ``footprint`` parameter of the functions ``binary_erosion``, ``binary_dilation``, ``binary_opening``, ``binary_closing``, ``erosion``, ``dilation``, ``opening``, ``closing``, ``white_tophat``, and ``black_tophat`` in ``skimage.morphology`` now accepts a tuple of a footprint and the number of times it should be applied iteratively.
+  (`#5482 <https://github.com/scikit-image/scikit-image/pull/5482>`_, `#6151 <https://github.com/scikit-image/scikit-image/pull/6151>`_)
+- Added support for the Modified Hausdorff Distance (MHD) metric in ``skimage.metrics.hausdorff_distance`` which performs better than the directed Hausdorff Distance (HD).
+  (`#5581 <https://github.com/scikit-image/scikit-image/pull/5581>`_)
+- Added support for the Modified Hausdorff Distance (MHD) metric in ``skimage.metrics.hausdorff_distance`` via the new parameter ``method`` .
+  (`#5581 <https://github.com/scikit-image/scikit-image/pull/5581>`_)
+- Added two datasets ``skimage.data.protein_transport`` and ``skimage.data.nickel_solidification``. (`#6087 <https://github.com/scikit-image/scikit-image/pull/6087>`_)
```

Comment:
```suggestion
- Added two datasets ``skimage.data.protein_transport`` and ``skimage.data.nickel_solidification`` (`#6087 <https://github.com/scikit-image/scikit-image/pull/6087>`_).
```

Reply:
On the danger of bikeshedding, I'm emulating NumPy here, e.g. [v1.23.0](https://github.com/numpy/numpy/releases/tag/v1.23.0). And I would argue that the referenced PRs are not part of the last sentence but rather an addendum of the whole bullet point.